tests/resources/transaction/test_pushtx.py

```diff
@@ -56,7 +56,14 @@ def test_get(self):
         tx.parents = self.manager.get_new_tx_parents(tx.timestamp)
         tx.resolve()
 
-        response = yield self.web.get('push_tx', {b'hex_tx': bytes(tx.get_struct().hex(), 'utf-8')})
+        push_tx_fn = self.web.post if is_post else self.web.get
+        hex_param = 'hex_tx' if is_post else b'hex_tx'
```

Comment:
Why don't we call a function that converts all keys and values to binary if needed?

Reply:
As discussed privately we decided to keep like this